server/models/TODOItem.model.js

```diff
@@ -0,0 +1,60 @@
+const mongoose = require('mongoose');
+require('mongoose-type-url');
+const Joi = require('joi');
+
+// new attribute to TODOItem is category
+const TODOItemSchema = new mongoose.Schema({
+  title: {
+    type: String,
+    lowercase: true,
+    required: true,
+    trim: true,
+  },
+  startDate: {
+    type: String,
+    required: true
```

Comment:
I also just remembered: weren't we going to make start and end dates for TODOs optional? Your point about how some people just want a list without times is definitely true but it appears to be mandatory.

Reply:
I did not say that start/end times should be optional. My initial point was simply whether we need to keep the time info for the date. We ended up agreeing to keep the time info.

Reply:
I'm fairly certain that it was you who brought up the point about using the TODO list just as a list as opposed to also using it to schedule tasks and that we decided that we would just make the dates optional so as to offer both. Ultimately this is your functionality though so it is up to you whether or not you want to do so. I just need the dates and times for the Gantt Chart.

Reply:
Nope. I am fairly certain I agreed with you guys that dates should be mandatory for TODO items. I was just arguing for the Dates to exclude times, since I thought that TODOs would mainly for used for tasks with faraway deadlines.